includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -89,6 +99,8 @@ public function get_settings(): WP_REST_Response {
 				'enabled_payment_method_ids'   => $this->wcpay_gateway->get_upe_enabled_payment_method_ids(),
 				'available_payment_method_ids' => $this->wcpay_gateway->get_upe_available_payment_methods(),
 				'is_wcpay_enabled'             => $this->wcpay_gateway->is_enabled(),
+				'is_manual_capture_enabled'    => $this->wcpay_gateway->get_option( 'manual_capture' ),
```

Comment:
It looks like this should return a boolean. The values saved currently in the DB are `'yes'` or `'no'` (rather than boolean):
- https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payment-gateway-wcpay.php#L1288
- https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payment-gateway-wcpay.php#L152

```suggestion
				'is_manual_capture_enabled'    => $this->wcpay_gateway->get_option( 'manual_capture' ) === 'yes' ? true : false,
```

Reply:
Thanks for the suggestion! I'm gonna edit a bit your suggested change (yoda condition) but yeah, you're absolutely right!